ibm_mq/datadog_checks/ibm_mq/metrics.py

```diff
@@ -56,10 +72,36 @@ def channel_metrics():
     return {
         'batch_size': pymqi.CMQCFC.MQIACH_BATCH_SIZE,
         'batch_interval': pymqi.CMQCFC.MQIACH_BATCH_INTERVAL,
-        'long_retry_count': pymqi.CMQCFC.MQIACH_LONG_RETRY,
-        'long_retry_interval': pymqi.CMQCFC.MQIACH_LONG_TIMER,
+        'long_retry': pymqi.CMQCFC.MQIACH_LONG_RETRY,
+        'long_timer': pymqi.CMQCFC.MQIACH_LONG_TIMER,
```

Comment:
Note to reviewers: Channel metrics where not submitted previously, so it's ok to rename them.

Reply:
I think the originals are more descriptive

Reply:
Most/all names match the name of the IBM MQ lib constant, should we keep that way for consistency?

Reply:
Ok, that makes sense but is this a breaking change?

Reply:
@hithwen My initial comment was this :) 

> Note to reviewers: Channel metrics where not submitted previously, so it's ok to rename them.